.devcontainer/docker-compose-vscode.yml

```diff
@@ -8,6 +8,7 @@ services:
     volumes:
       # This is where VS Code should expect to find your project's source code and the value of "workspaceFolder" in .devcontainer/devcontainer.json
       - ..:/workspace:cached
+      - ./logs:/var/log/tmx
```

Comment:
How come we only added this to the dev container setup. Do we not need it in the regular setup.

Reply:
Already exists in the regular setup. It was missing here.